argocd-boostrap/argocd-bootstrap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-cm
+data:
+  repositories: |
+    - url: https://github.com/helm/charts.git
```

Comment:
Why do we need this config map? I would like to pass these values in the constructor to the Addon. You can generate a config map, but it would be easier to just pass it over as env variables.

Reply:
Agreed. will remove this in my next commit